resources/resincli.robot

```diff
@@ -157,6 +162,11 @@ Get the last git commit from "${directory}"
     Process ${result}
     [Return]    ${result.stdout}
 
+Check that "${interface}" ip address is not in the ip address field returned by the "${device_uuid}" API
+    ${ip_address} =    Get "${interface}" ip address using socket "unix\#/tmp/console.sock"
```

Comment:
Check that "${device_uuid}" does not return "${resin-vpn}" IP address through API